test/utils/TestUtils.ts

```diff
@@ -0,0 +1,81 @@
+export function bytes32(n: number): string {
+    return '0x' + n.toString().repeat(64).slice(0, 64);
+}
+
+export type IntString = string;
```

Comment:
If you want to enforce integer in a string then perhaps you would like to use
``  export type IntString = `${number}` ``

Reply:
It's not used this type, so I removed it.